### DISCUSSION TITLE
Authorization error via google after changing email in google

### DISCUSSION THREAD

**Discussion by mr-bantsevich** [outline/outline#4380] [Self Hosting]
I have fresh version: 0.66.2

I have authorization only through google account. I changed my email in google from max@domain.name to new max@domain_new.name.
And after that I can't enter.

<img width="927" alt="Снимок экрана 2022-11-02 в 22 19 10" src="https://user-images.githubusercontent.com/12588034/199614105-cde90fd1-a5f4-4fcb-83e8-0a7b6db6ba7b.png">

**Comment by tommoor** [comment]
You need to change the domain in the `authorizations` table to point to your new domain, this is not considered a bug.

**Reply by mr-bantsevich** [reply]
Yes, sure, I already did it. We have 2 domains.
So, new users can enter from both domains.
